src/constructors.jl

```diff
@@ -68,15 +75,21 @@ for T in (Unsigned, Integer, Base.GMP.CdoubleMax)
         end
     end
 end
+function Acb(x::Arf; prec::Integer = precision(x))
+    res = Acb(prec = prec)
+    # There is not set! with Acb and Arf. So create intermediate Arb :shrug:
+    set!(res, Arb(x, prec = prec))
+    return res
+end
```

Comment:
If we want to microoptimize we could construct only an `arb_struct`, though from Julia version 1.5 this might be optimized away anyway.

Reply:
I don't think this conversion happens that often at all.